src/applications/vaos/reducers/expressCare.js

```diff
@@ -0,0 +1,109 @@
+import moment from 'moment';
+
+import {
+  FETCH_EXPRESS_CARE_WINDOWS,
+  FETCH_EXPRESS_CARE_WINDOWS_FAILED,
+  FETCH_EXPRESS_CARE_WINDOWS_SUCCEEDED,
+} from '../actions/expressCare';
+
+import { FETCH_STATUS } from '../utils/constants';
+
+const initialState = {
+  windowsStatus: FETCH_STATUS.notStarted,
+  allowRequests: false,
+  localWindowString: null,
+  minStart: null,
+  maxEnd: null,
+};
+
+export default function expressCareReducer(state = initialState, action) {
+  switch (action.type) {
```

Comment:
What do you think about also copying over the form page opened/updated reducers?